softioc/asyncio_dispatcher.py

```diff
@@ -74,6 +87,9 @@ async def async_wrapper():
                     await ret
                 if completion:
                     completion(*completion_args)
+            except asyncio.CancelledError:
+                print("Coroutine was cancelled")
+                raise
```

Comment:
Please remove this clause - the `logging.exception` line below should be all the output we need.